processor/fluentbitk8sprocessor/README.md

```diff
@@ -0,0 +1,9 @@
+## <a name="fluentbitk8sprocessor"></a>Fluent bit k8s rocessor
```

Comment:
Is this processor specific to sumologic-kubernetes-collection?

Reply:
So in order to perform metadata on the logs from the fluentbit from the k8s we need to apply that plugin. I believe it can be useful for the community too, eventually the functionality can be moved to the fluentforwarder, but I think it's better as separated processor